runtime/databricks/automl_runtime/forecast/prophet/model.py

```diff
@@ -51,8 +52,7 @@
         {
             "pip": [
                 f"prophet=={prophet.__version__}",
-                f"cloudpickle=={cloudpickle.__version__}",
-                f"databricks-automl-runtime==0.2.5"
+                f"cloudpickle=={cloudpickle.__version__}"
```

Comment:
Just for my understanding. What exactly is needed to be included in this conda_env used by `mlflow.pyfunc.log_model`?

Reply:
This environment is for model serving